scalajvm/app/controllers/api/Quotes.scala

```diff
@@ -1,30 +1,47 @@
 package controllers.api
 
-import helpers.ActionWithTx
+import helpers.{ActionWithTx, Cors, RequestWithSession}
 import models.queries.{QuoteFilter, QuoteOrdering, QuoteQueries}
 import models.data.Quote
 import play.api.mvc._
 import ru.org.codingteam.loglist.dto.QuoteDTO
 import ru.org.codingteam.loglist.QuoteCount
 
 object Quotes extends Controller {
-  def getQuote(id: Long) = ActionWithTx { request =>
+  private def action(responseAction: RequestWithSession[AnyContent] => Result) = ActionWithTx { request =>
```

Comment:
I have already done that in scope of #94, so I think I'll just merge these pull requests together, and review the result afterwards.